### ISSUE TITLE
documentation for API workflow

### ISSUE THREAD

**Issue by isabelizimm** [rstudio/rsconnect-python#241]
With the removal of the API docs, it would be good for users to have some sort of documentation to show how to use `rsconnect-python` API in a narrative way.

More than happy to start a PR of a markdown/Quarto file for the bits that I tend to use a lot (`RSConnectServer`, `deploy_python_fastapi`, etc) to start this off! LMK what format is most useful to you all.

**Comment by mmarchetti** [comment]
Yes, I agree completely. We've been thinking that a how-to (vs. a reference doc) would be the best way to go here. The docs are all contained in the repo README.md file.
